Package.swift

```diff
@@ -96,10 +95,10 @@ let targets: [Target] = [
         dependencies: [
             "TuistKit",
             "TuistSupport",
-            "Path",
             "TuistLoader",
             "ProjectDescription",
             "ProjectAutomation",
+            pathDependency,
```

Comment:
Not stricly related to the warning fix but it was the only other place that Path was included via the `"Path"` syntax.